modules/sdp/SdpSimulcast.ts

```diff
@@ -5,7 +5,7 @@ import { MediaType } from '../../service/RTC/MediaType';
 import { SIM_LAYERS, SSRC_GROUP_SEMANTICS } from '../../service/RTC/StandardVideoQualitySettings';
 
 
-interface IDescription {
+export interface IDescription extends RTCSessionDescription {
```

Comment:
Can you get rid of this and use `RTCSessionDescription` instead of `IDescription`?